### DISCUSSION TITLE
Release 5.0.6 discussions

### DISCUSSION THREAD

**Discussion by butlerpd** [SasView/sasview#2400] [General]
Rather than a free for all editing the wiki, I am opening a discussion so that we can more easily discuss prioritizations, what might be too dangerous to include, might be inappropriate to include, should (or should not) be a "MUST HAVE" etc.

**Comment by rozyczko** [comment]
I think bug fixes, which haven't yet been marked as MUST HAVE, should be prioritized.
E.g.
https://github.com/SasView/sasview/pull/2099/commits/eb34b3152dfb2151004859eda674606aa457e1e3
https://github.com/SasView/sasview/pull/2362/commits/1d4ead940a8d46e01f9c6b4b0eb9d6fb5a617cff
https://github.com/SasView/sasview/pull/2320/commits/a8bac2421fbe6f74d567a3895cce41e5c0537bab
https://github.com/SasView/sasview/pull/2302/commits/7610f150ba27af06300c5b19186516acbecb50f4

Also, QoL changes should probably be added as they have some impact on how users perceive the application.
https://github.com/SasView/sasview/pull/2275/commits
https://github.com/SasView/sasview/pull/2273/commits/92580782682bc01ec386e14e32a490133d33b81a
https://github.com/SasView/sasview/pull/2329/commits/0b9189d08e10f6ff2d7c663ec2cf3997d5fd5d01

**Comment by krzywon** [comment]
https://github.com/SasView/sasview/commit/a8bac2421fbe6f74d567a3895cce41e5c0537bab is a part of #2320 which was a fix for copy/paste that was changed #2210. None of this work is present in 5.0.5 so it shouldn't be included. The same can be said for the Reports changes in #2275, but we should definitely include the rest of the changes there because of the improved loading time.
 
The remaining fixes you suggested are listed at https://github.com/SasView/sasview/wiki/v5.0.6. If you think they are high priority, update the wiki page and label them as such.
